src/modules/hyprland/workspaces.cpp

```diff
@@ -625,10 +670,13 @@ void Workspaces::init() {
 
   for (Json::Value workspaceJson : workspacesJson) {
     std::string workspaceName = workspaceJson["name"].asString();
+    spdlog::info("initing workpsace {}:{}", workspaceJson["id"], workspaceJson["name"]);
```

Comment:
```suggestion
    spdlog::debug("initing workspace {}:{}", workspaceJson["id"], workspaceJson["name"]);
```

Reply:
Oh, honestly that's a leftover; didn't intend to log anything haha. Good catch

Reply:
Addressed in 4339030